heft-plugins/heft-sass-plugin/src/SassProcessor.ts

```diff
@@ -1,11 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
+/// <reference lib="dom" />
```

Comment:
What needs this?

Reply:
I added this because type declaration of dart-sass refers to the dom URL. At first, I considered adding it directly to tsconfig, but I think it's worse to affect the whole because the project itself is for the node environment. welcome your suggestion if you have any other comments or better ideas